smart-metastore/src/main/java/org/smartdata/metastore/dao/AccessCountDao.java

```diff
@@ -171,10 +173,39 @@ public void createProportionTable(AccessCountTable dest, AccessCountTable source
 
   public void updateFid(long fidSrc, long fidDest) {
     JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
-    for (AccessCountTable table : getAllSortedTables()) {
-      String sql = String.format("update %s set %s=%s where %s=%s", table.getTableName(),
-          AccessCountDao.FILE_FIELD, fidDest, AccessCountDao.FILE_FIELD, fidSrc);
-      jdbcTemplate.execute(sql);
```

Comment:
I am thinking whether we can just catch exception thrown by `jdbcTemplate.execute(sql)` and ignore the exception considering no huge impact caused.

Reply:
#2203 seems to be a better solution. I will test that patch.